src/Common/Thermodynamics/relations.jl

```diff
@@ -908,7 +908,7 @@ saturation_excess(ts::ThermodynamicState) = saturation_excess(
 
 Condensate of the phase partition.
 """
-condensate(q::PhasePartition) = q.liq + q.ice
+condensate(q::PhasePartition) = max(0, q.liq) + max(0, q.ice)
```

Comment:
Maybe better `max(0, q.liq + q.ice)`? (And perhaps also make it a `Thermodynamics` function.)